apps/extension/src/pages/earn/overview/tutorial-modal/earn-claim-content.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { FunctionComponent, Fragment } from "react";
+import { FormattedMessage } from "react-intl";
+import { XAxis } from "../../../../components/axis";
+import { Box } from "../../../../components/box";
+import { Button } from "../../../../components/button";
+import { Gutter } from "../../../../components/gutter";
+import {
+  H4,
+  Body3,
+  Subtitle4,
+  Button2,
+} from "../../../../components/typography";
+import { ColorPalette } from "../../../../styles";
+import { Image } from "../../../../components/image";
+
+export const EarnClaimContent: FunctionComponent<{
+  tokenName: string;
+  moveToNext: () => void;
+  moveToPrev: () => void;
+}> = ({ tokenName: tokenName, moveToNext, moveToPrev }) => (
```

Comment:
```suggestion
}> = ({ tokenName, moveToNext, moveToPrev }) => (
```
문제 될 건 없지만- 똑같은 이름이어서 지우는게 좋을 것 같아요